### PR TITLE
close out dusko grant

### PR DESCRIPTION
**Pull request by thelifeandtimes**

### CODE REVIEW

File: content/grants/dusko-valve.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ assignee = "~tasset-norlux"
 champion = "~tacryt-socryp"
 grant_id = "P0143"
 completed = false
-canceled = false
+canceled = true
```
Comment (mikeosborne) [inline content/grants/dusko-valve.md]:
```suggestion
canceled = false
```

File: content/grants/dusko-valve.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ assignee = "~tasset-norlux"
 champion = "~tacryt-socryp"
 grant_id = "P0143"
 completed = false
```
Comment (mikeosborne) [inline content/grants/dusko-valve.md]: ### 
```suggestion
completed = true
```

**Comment by mikeosborne** [comment]
I thought we decided that a project with at least one completed milestone would be marked "complete = true"

**Comment by thelifeandtimes** [comment]
yup, that works. Just wasn't following how those play out here. should be gtg now.

**Review by mikeosborne — APPROVED** [review]

**Pull request merged**